sotodlib/tod_ops/fft_ops.py

```diff
@@ -454,3 +454,76 @@ def fit_noise_model(
     if merge_fit:
         aman.wrap(merge_name, noise_fit_stats)
     return noise_fit_stats
+
+
+def build_filter_params_dict(
+    filter_name,
+    noise_fit=None,
+    filter_params=None
+):
+    """
+    Build the filter parameter dictionary from a provided
+    dictionary or from noise fit results.
+
+    Args
+    ----
+    filter_name : str
+        Name of the filter to build the parameter dict for.
+    noise_fit: nparray
+        Array of noise model fit parameters sized nparams x ndets.
```

Comment:
It looks like is actually expected to be the `AxisManager` returned from `tod_ops.fft_ops.fit_noise_model` not an nparray.

Reply:
Exact. Changed the docstring in the function